app/models/work_package/pdf_exporter.rb

```diff
@@ -420,15 +429,27 @@ def textstring(s)
     def fix_text_encoding(txt)
       # these quotation marks are not correctly rendered in the pdf
       txt = txt.gsub(/[â€œâ€�]/, '"') if txt
+      encoding = case current_language.to_s.downcase
+        when 'ko'    then "CP949"
+        when 'ja'    then "CP932"
+        when 'zh'    then "gb18030"
+        when 'zh-tw' then "Big5"
```

Comment:
Indent `when` as deep as `case`.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.